src/features/content-answers/__tests__/Answer.test.tsx

```diff
@@ -6,17 +6,19 @@ import Answer from '../Answer';
 describe('features/content-answers/answer', () => {
     const renderComponent = (props: { answer: string; isLoading: boolean }) => render(<Answer {...props} />);
 
-    test('should render the answer', () => {
+    test('should only render the answer', () => {
         renderComponent({ answer: 'some answer', isLoading: false });
 
         const answer = screen.getByText('some answer');
         expect(answer).toBeInTheDocument();
+        expect(screen.queryByTestId('LoadingElement')).not.toBeInTheDocument();
     });
 
-    test('should render the loading element', () => {
+    test('should only render the Loading Element', () => {
         renderComponent({ answer: '', isLoading: true });
 
         const loadingElement = screen.getByTestId('LoadingElement');
         expect(loadingElement).toBeInTheDocument();
+        expect(screen.queryAllByTestId('content-answers-grid-card')).toHaveLength(1);
```

Comment:
I think this should assert if the answer is there not the card, its easier to understand in terms of readability. noone is going to know that the card is actually under the LoadingElement too unless they check in that component too

Reply:
since the answer prop here is given as ''. I think it does not make sense to check if text '' not in the document. Instead, I checked length of children list under Answer component is 1 which should be loading element.